modules/savanna/modules/workgroup-workspace/pages/workgroups/workspace-logs.adoc

```diff
@@ -0,0 +1,43 @@
+= Log
+:experimental:
+
+Log provides full-text search capabilities for workspace logs (including historical logs, with a default retrievable period of 30 days) and allows filtering of logs based on time, nodes, and components. By utilizing log, teams can not only quickly locate and resolve issues but also gain in-depth insights into system performance, user behavior, and operational auditing, ultimately enhancing system reliability and performance.
+
+[TIP]
+====
+The Log feature is only available to enterprise-level customers. If you need to use this feature, please contact support@tigergraph.com to enable it for your organization.
+====
+
+== How to Enable Log
+
+.Enable log when creating a workgroup:
+
+. When you xref:savanna:workgroup-workspace:workgroups/how2-create-a-workgroup.adoc[create a workgroup], you can check the option to enable log collection on the page.
++
+image::enablelogwhencreateworkgroup.png[width="1600"]
+
+.Enable log when after workgroup is created:
```

Comment:
```suggestion
```
This dot-sentence is both redundant and interfering with the correct numbering on your page.